UNRELEASED.md

```diff
@@ -60,6 +60,19 @@ following warning will be shown during compilation or in the IDE:
 
 For more information, see [Reference: Templates: Template-local Definitions (Deprecated)](https://docs.daml.com/2.8.0/daml/reference/templates.html#template-local-definitions-deprecated)
 
+## Removal of deprecated 'controller..can' syntax
+
+The 'controller..can' syntax for defining template choices, deprecated since
+Daml 2.0, has now been completely removed, so projects that use them will no
+longer be accepted. Those choices should instead be defined using 'choice-first'
```

Comment:
```suggestion
The 'controller..can' syntax for defining template choices, deprecated since
Daml 2.0, has now been completely removed. Projects that use this syntax are no
longer accepted. Those choices should instead be defined using 'choice-first'
```

Reply:
thanks!